core/language/en-GB/Libraries.multids

```diff
@@ -0,0 +1,6 @@
+title: $:/language/Libraries/
```

Comment:
There are a few implications of adding a new language file, in particular the translator edition will need changing. What's the motivation here?

Reply:
I thought, it would be a cleaner structure, since the library texts where in `Misc.multids`

But we could wait until we get more 3rd party library configs.